repository-data/application/src/main/resources/hcm-config/configuration/update/DW-343-GdprSummaryOrder.groovy

```diff
@@ -0,0 +1,48 @@
+package org.hippoecm.frontend.plugins.cms.admin.updater
+
+import org.apache.jackrabbit.value.StringValue
+import org.onehippo.repository.update.BaseNodeUpdateVisitor
+
+import javax.jcr.Node
+import javax.jcr.Value
+/**
+ * This migration causes the search rank to be set to 1 to push these docs to the top of
+ * 'relevance' searches
```

Comment:
Does this comment need updating?

Reply:
Good point, it comes from a totally different groovy script :)